Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -2793,8 +2793,15 @@ class WebPage final : public API::ObjectImpl<API::Object::Type::BundlePage>, pub
     bool m_completingSyntheticClick { false };
     bool m_hasHandledSyntheticClick { false };
 
-    enum SelectionAnchor { Start, End };
-    SelectionAnchor m_selectionAnchor { Start };
+    enum class SelectionAnchor : bool { Start, End };
+    SelectionAnchor m_selectionAnchor { SelectionAnchor::Start };
+
+    enum class SelectionFlippingState : uint8_t {
```

Comment:
Maybe we should add Bidi prefix to make it clear this is for bidi?

Reply:
Sounds good, will rename to `BidiSelectionFlippingState`.